spt/OrangeBox/patterns.hpp

```diff
@@ -355,22 +363,6 @@ namespace Patterns
 			"xxxxxxxxxxxx????xxxxxxxxxxxxxxxxxx"
 		},
 
-		{
```

Comment:
Why was this removed?

Reply:
I have no idea.

Reply:
Add it back?

Reply:
After further examination, I've come to the conclusion that I removed it because the same two patterns are directly above it.

Reply:
Fair enough.